### ISSUE TITLE
Support for vertically/horizontally maximized windows

### ISSUE THREAD

**Issue by viliampucik** [germanfr/cinnamon-transparent-panels#39]
Please, would it be possible to add support for vertically (horizontally) maximized windows to change top/bottom (left/right) panel transparency as well?

Currently the panel transparency is affected only by fully maximized windows. However I have noticed that in Budgie desktop environment the panel transparency is affected by partially maximized windows and it is a very eye pleasing :)

**Comment by germanfr** [comment]
Yes, there were plans for this. The code architecture is just prepared for it. But unfortunately I don't have Cinnamon installed currently, so I won't be developing new features in the near future, only bug fixes.

I'll leave this open anyways just in case. Thanks!

**Comment by viliampucik** [comment]
Thanks for the clarification @germanfr, appreciated! :)

Please, may I ask what do you use instead of Cinnamon these days?

**Comment by germanfr** [comment]
GNOME. I prefer Cinnamon the desktop and the extension system is much better, but the whole GNOME environment works better for me.
